terraform/aks/.terraform.lock.hcl

```diff
@@ -0,0 +1,82 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
✂️

Reply:
Oof. Thanks. I'll add it to the gitignore too 🤦‍♂️